tools/Dockerfile.ubuntu18.04.install

```diff
@@ -0,0 +1,60 @@
+# ==============================================================================
+#  Copyright 2019 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# ==============================================================================
+
+ARG http_proxy
+ARG https_proxy
+ARG base_image
+ARG ngtf_build_options
+
+FROM $base_image as base
+ARG http_proxy
+ARG https_proxy
+ARG ngtf_build_options
+
+RUN git clone https://github.com/tensorflow/ngraph-bridge.git /src/ngraph-bridge
```

Comment:
we don't have `src` anymore. it will be just `ngraph-bridge/ngraph_bridge`

Reply:
I removed the `/src`prefix in PR #355 . Please switch over to that PR which resolves the CLA problem.

Reply:
So, is this PR a duplicate? Can we close this?